src/cargo/core/compiler/mod.rs

```diff
@@ -764,6 +765,14 @@ fn add_error_format_and_color(
     } else {
         let mut color = true;
         match cx.bcx.build_config.message_format {
+            MessageFormat::Human if nightly_features_allowed() => {
```

Comment:
I noticed this changed it to unconditionally enable it on nightly.  We haven't really done that before, but I don't really have an objection to it.  We have problems with figuring out how to get people to test unstable flags, so unconditionally enabling it on nightly might be a good idea, since it is just a visual change. I'd like to hear from other cargo members, though.

Reply:
Sorry for the late repsponse, but I think I'd prefer if this still needed to be opted-in to if that's ok, because otherwise if `-Zterminal-width` changes it'd break all nightly users by default until we can ship a fix

Reply:
Fair enough.